regolith/helpers/a_projectumhelper.py

```diff
@@ -177,8 +180,197 @@ def db_updater(self):
+            ("Check db entry", "CAREFULLY double and triple check the meta-data associated with the paper in citations.yml:"),
+            ("Check grants in the db entry", "grant{} lists just the billinge-group grants that appeared in the acknowledgement section. They have standard abbreviations that are listed at the top of the citations.yml file, e.g., fwp, EFRC10, etc.. Use the right standard or the whole system becomes broken! If not sure.....ask Simon. List all grants in a comma-separated list."),
+            ("Check the facility in the db entry", "facility{} is every beamline that was used for data collection. Again, use the standard abbreviations at the top of the file. Use two levels of granularity for each, so X17A would be: 'nsls, x17a', if X17A and X7B were used it would be 'nsls, x17a, x7b' and so on."),
+            ("Check the nb in the db entry", "nb{} is some other tags, also listed at the top of the file. 'art' for a regular article and 'hilite' if it is one of our top top papers are the most common."),
+            ("Check db errors", "In your rg-db-public/local directory, run `regolith build publist --people lyang` (replace lyang with your name) to make sure that you publist is building properly. Make sure that the publication appears correctly with no errors and fix anything. If there are problems with the latex building, run the commands with --no-pdf, which yields the latex source but doesn't build it, then build the latex manually."),
+            ("Add to Zotero", "Add the published reference to the billinge-group-bib folder in our group Zotero account"),
+            ("Finish the highlight slide", "Make a highlight of your work and put it in gitlab/highlights (if not done already during the accepted paper checklist). Look in there for standards to work from. This is an important activity. Now you have done your great work, this is how you can advertise it to others. Top papers we send these highlights to the funding agencies.  Iterate the highlight with Simon till it is converged."),
+            ("Make figures and talk slides", "Make figures and talk slides that will be used in talks and place these on gitlab on talks/figures.  Iterate this with Simon till it is converged."),
+            ("Update arXiv if necessary", "If the paper was listed on a preprint server like arXiv, submit a note to arXiv that the paper has appeared and give the full reference. If the journal copyright allows you can post the published version here, but normally that is not alllowed! Still, it is important that people who find the paper on arXiv get directed to the correct reference."),
+            ("Ask Simon if anything unfinished", "Ask Simon about any items unfinished."),
+            ("Email Simon", "Email Simon if finish the above."),
+        ]
+
+        checklistm_list = []
+        for name, objective in presubmission_checklist:
+            checklistm = {'due_date': now + relativedelta(days=7),
```

Comment:
haha, this is nice.  I actually meant the for loops to remove copy-pasted boilerplate though :).  All of these ``for name, objective`` blocks are identical to each other except for the due date and the name of the stage, so it could look something like:

```
checklistm =  {'objective': objective,
                       'audience': [],
                       'notes': [],
                       'status': 'converged',
                       'type': 'pr'
                       }
for list_name, delay_days in zip(checklists, checklist_delay_days):
        checklistm.update({'due_date': now + relativedelta(days=delay_days),
                                        'name': f"{list_name} - {name}")
```
sthg like that.  As time goes on we have more and more code to maintain so developing habits like this that propagate forward in this and other code can really save us lots of bug fixing in the future.....